Cargo.toml

```diff
@@ -17,6 +17,9 @@ license = "GPL-3.0-only"
 edition = "2021"
 
 [workspace.dependencies]
+common = { path = "./common" }
+streaming-types = { path = "./streaming-types" }
```

Comment:
```suggestion
supermusr-streaming-types = { path = "./streaming-types" }
```

Reply:
You'll also need to rename these in the crates that inherit the dependency.

Reply:
Sorry, I've just noticed that as well.